_tests_/measureReportCompare.ts

```diff
@@ -0,0 +1,132 @@
+/**
```

Comment:
This file should be moved to a different branch for the follow on work to run against the fixture data.